Redis CLI test cases failed in Ubuntu 24.04

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.


### Version

unstable

### Minimal reproduce step

https://github.com/apache/kvrocks/actions/runs/9925485209/job/27418202135?pr=2409
```
--- FAIL: TestRedisCli (6.59s)
    --- FAIL: TestRedisCli/test_interactive_cli (0.00s)
        --- FAIL: TestRedisCli/test_interactive_cli/INFO_response_should_be_printed_raw (0.00s)
            cli_test.go:167: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:167
                	Error:      	Received unexpected error:
                	            	EOF
                	Test:       	TestRedisCli/test_interactive_cli/INFO_response_should_be_printed_raw
        --- FAIL: TestRedisCli/test_interactive_cli/Status_reply (0.00s)
            cli_test.go:108: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:108
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:174
                	Error:      	Received unexpected error:
                	            	write |1: broken pipe
                	Test:       	TestRedisCli/test_interactive_cli/Status_reply
        --- FAIL: TestRedisCli/test_interactive_cli/Integer_reply (0.00s)
            cli_test.go:108: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:108
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:178
                	Error:      	Received unexpected error:
                	            	write |1: broken pipe
                	Test:       	TestRedisCli/test_interactive_cli/Integer_reply
        --- FAIL: TestRedisCli/test_interactive_cli/Bulk_reply (0.00s)
            cli_test.go:108: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:108
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:183
                	Error:      	Received unexpected error:
                	            	write |1: broken pipe
                	Test:       	TestRedisCli/test_interactive_cli/Bulk_reply
        --- FAIL: TestRedisCli/test_interactive_cli/Multi-bulk_reply (0.00s)
            cli_test.go:108: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:108
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:189
                	Error:      	Received unexpected error:
                	            	write |1: broken pipe
                	Test:       	TestRedisCli/test_interactive_cli/Multi-bulk_reply
        --- FAIL: TestRedisCli/test_interactive_cli/Parsing_quotes (0.00s)
            cli_test.go:108: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:108
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:196
                	Error:      	Received unexpected error:
                	            	write |1: broken pipe
                	Test:       	TestRedisCli/test_interactive_cli/Parsing_quotes
        cli_test.go:162: 
            	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:162
            	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:213
            	Error:      	Error message not equal:
            	            	expected: "signal: terminated"
            	            	actual  : "exit status 127"
            	Test:       	TestRedisCli/test_interactive_cli
    --- FAIL: TestRedisCli/test_tty_cli (0.01s)
        --- FAIL: TestRedisCli/test_tty_cli/Status_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:220
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_tty_cli/Status_reply
        --- FAIL: TestRedisCli/test_tty_cli/Integer_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:226
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_tty_cli/Integer_reply
        --- FAIL: TestRedisCli/test_tty_cli/Bulk_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:231
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_tty_cli/Bulk_reply
        --- FAIL: TestRedisCli/test_tty_cli/Multi-bulk_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:2[42](https://github.com/apache/kvrocks/actions/runs/9925485209/job/27418202135?pr=2409#step:16:43)
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_tty_cli/Multi-bulk_reply
        --- FAIL: TestRedisCli/test_tty_cli/Read_last_argument_from_pipe (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:2[51](https://github.com/apache/kvrocks/actions/runs/9925485209/job/27418202135?pr=2409#step:16:52)
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_tty_cli/Read_last_argument_from_pipe
        --- FAIL: TestRedisCli/test_tty_cli/Read_last_argument_from_file (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:264
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_tty_cli/Read_last_argument_from_file
    --- FAIL: TestRedisCli/test_notty_cli (0.01s)
        --- FAIL: TestRedisCli/test_notty_cli/Status_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:271
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Status_reply
        --- FAIL: TestRedisCli/test_notty_cli/Integer_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:277
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Integer_reply
        --- FAIL: TestRedisCli/test_notty_cli/Bulk_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:282
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Bulk_reply
        --- FAIL: TestRedisCli/test_notty_cli/Multi-bulk_reply (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:289
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Multi-bulk_reply
        --- FAIL: TestRedisCli/test_notty_cli/Quoted_input_arguments (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:294
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Quoted_input_arguments
        --- FAIL: TestRedisCli/test_notty_cli/No_accidental_unquoting_of_input_arguments (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:298
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/No_accidental_unquoting_of_input_arguments
        --- FAIL: TestRedisCli/test_notty_cli/Read_last_argument_from_pipe (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:316
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Read_last_argument_from_pipe
        --- FAIL: TestRedisCli/test_notty_cli/Read_last_argument_from_file (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:329
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/test_notty_cli/Read_last_argument_from_file
    --- FAIL: TestRedisCli/Scan_mode (0.00s)
        cli_test.go:122: 
            	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
            	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:339
            	Error:      	Received unexpected error:
            	            	exit status 127
            	Test:       	TestRedisCli/Scan_mode
    --- FAIL: TestRedisCli/Piping_raw_protocol (5.[54](https://github.com/apache/kvrocks/actions/runs/9925485209/job/27418202135?pr=2409#step:16:55)s)
        cli_test.go:122: 
            	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
            	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:370
            	Error:      	Received unexpected error:
            	            	exit status 127
            	Test:       	TestRedisCli/Piping_raw_protocol
    --- FAIL: TestRedisCli/Test_redis-cursor-compatible_mode (0.01s)
        --- FAIL: TestRedisCli/Test_redis-cursor-compatible_mode/Use_reids-cli_--bigkeys (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:382
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/Test_redis-cursor-compatible_mode/Use_reids-cli_--bigkeys
        --- FAIL: TestRedisCli/Test_redis-cursor-compatible_mode/Use_reids-cli_--memkeys (0.00s)
            cli_test.go:122: 
                	Error Trace:	/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:122
                	            				/home/runner/work/kvrocks/kvrocks/tests/gocase/integration/cli/cli_test.go:387
                	Error:      	Received unexpected error:
                	            	exit status 127
                	Test:       	TestRedisCli/Test_redis-cursor-compatible_mode/Use_reids-cli_--memkeys
```

### What did you expect to see?

--

### What did you see instead?

--

### Anything Else?

--

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

--- a/.github/workflows/kvrocks.yaml
+++ b/.github/workflows/kvrocks.yaml
@@ -150,13 +150,12 @@ jobs:
           - name: Ubuntu Clang
             os: ubuntu-22.04
             compiler: clang
-          # FIXME: https://github.com/apache/kvrocks/issues/2411
-          # - name: Ubuntu 24 GCC
-          #   os: ubuntu-24.04
-          #   compiler: gcc
-          # - name: Ubuntu 24 Clang
-          #   os: ubuntu-24.04
-          #   compiler: clang
+          - name: Ubuntu 24 GCC
+            os: ubuntu-24.04
+            compiler: gcc
+          - name: Ubuntu 24 Clang
+            os: ubuntu-24.04
+            compiler: clang
           - name: Ubuntu GCC ASan
             os: ubuntu-22.04
             without_jemalloc: -DDISABLE_JEMALLOC=ON

